win/pkgs/env/toolchain.ps1

```diff
@@ -169,8 +169,12 @@ if (-Not $Env:ROASTER_TOOLCHAIN_COMMITED)
 
     if (${Env:VSCMD_VER} -eq $null)
     {
-        ${VS_HOME} = & "${Env:ProgramFiles(x86)}/Microsoft Visual Studio/Installer/vswhere.exe" -latest -property installationPath
-        Invoke-Expression $($(cmd /c "`"${VS_HOME}/VC/Auxiliary/Build/vcvarsall.bat`" x64 10.0.16299.0 & set") -Match '^.+=' -Replace '^','${Env:' -Replace '=','}="' -Replace '$','"' | Out-String)
+        if (${Env:VS_HOME} -eq $null)
```

Comment:
I don't want to give user any chance to set the wrong compiler.
If `vswhere` is not available, how about find it in registry?

Reply:
What do you mean by the wrong compiler? If my understanding is correct, the toolchain file need to locate the vs dev prompt script to initialize all environment variables. We can add some additional check to validate the MSVC version, SDK availability, etc.

Besiodes,Installing VC++ BuildTool is quite common in a dockernize scenario, [link](https://docs.microsoft.com/en-us/visualstudio/install/build-tools-container).

Reply:
Checked both VC Toolset version and WinSDK version in the script.